src/dashboard.py

```diff
@@ -1645,6 +1647,9 @@ def sqlUpdate(statement: str, paramters: tuple = ()) -> sqlite3.Cursor:
             sqldb.commit()
         except sqlite3.OperationalError as error:
             print("[WGDashboard] SQLite Error:" + str(error) + " | Statement: " + statement)
+        except Exception as e:
+            print("[WGDashboard] SQLite Error:" + str(error) + " | Statement: " + statement)
```

Comment:
Can't this also be caught with a general exception instead of Exception AND sqlite3.operationalError?

Reply:
Yes you're right haha.. Post-Christmas break syndrome ;)

Reply:
No worries! That's why the review is handy!